hazelcast/src/main/java/com/hazelcast/jet/impl/submitjob/memberside/JobMetaDataParameterObject.java

```diff
@@ -34,11 +34,16 @@ public class JobMetaDataParameterObject {
 
     private UUID sessionId;
 
+    private boolean directJobExecution;
+
+    private boolean deleteJarAfterExecution;
```

Comment:
Why do we need this? We can always derive the value from `directJobExecution`, right?

Reply:
Yes we can derive the meaning but I wanted them to be different explicitly. So that  when somebody asks for deleting the jar on the member later on, we can do it

Reply:
Then add a method with such name, but having 2 such dependent variables is confusing - I was first searching where the value comes from, only to realize it is derived from the other one.

Reply:
Moved the deletion to JobUploadStatus. Deleted deleteJarAfterExecution field
Rev : bb5b1fc1681799745afce0373d490a9846cdbd70